bootstrap/components/sinks/kafka-sink-topology-component.json

```diff
@@ -35,13 +35,23 @@
         "hint": "schema,override"
       },
       {
-        "uiName": "Writer schema version",
+        "uiName": "Writer Schema Branch",
+        "fieldName": "schemaBranch",
+        "isOptional": true,
+        "tooltip": "Schema branch to be used fetching for writer schema versions. Default branch is MASTER",
+        "type": "enumstring",
+        "options": [],
+        "hint": "dependOn-topic,schemaBranch",
+        "defaultValue": "MASTER"
+      },
+      {
+        "uiName": "Writer Schema Version",
         "fieldName": "writerSchemaVersion",
         "isOptional": false,
         "tooltip": "Version of schema for topic to use for serializing the message. Default is the latest version for the schema",
         "type": "enumstring",
         "options": [],
```

Comment:
@shahsank3t Can you also check the same for this? Meaning, if UI will work when a particular version of schema that user previously selected is not returned anymore in the list because it was deleted. In that case we need to make sure user is asked for an input again.

Reply:
@priyank5485 This is a mandatory field, so if the previously selected version is no longer available in the dropdown list, user will be forced to pick another one by the UI